apps/web/hooks/api/useUserData.ts

```diff
@@ -0,0 +1,13 @@
+import { useQuery } from '@tanstack/react-query';
+import { platformAPI } from '../../config';
```

Comment:
@DarkPhoenix2704  i cant acess ```@pages/folder``` even though i added it in tsconfig can you take a look at it ?